app/page.tsx

```diff
@@ -116,6 +147,10 @@ async function getASRouterLocalMessageInfoFromFile(): Promise<
   );
   let json_data = JSON.parse(data);
 
+  if (isLookerEnabled) {
```

Comment:
Please add a line to the backlog spreadsheet to indicate to the user/developer that the list is incomplete when looker is disabled.